### DISCUSSION TITLE
Channel private videos

### DISCUSSION THREAD

**Discussion by alibori** [alchemyguy/YoutubeLaravelApi#31] [Q&A] [answered]
Hi!

Is it possible to get the private videos of a channel?

Thank you!

**Comment by alchemyguy** [comment, marked as answer]
@alibori 
 yes, you can get all videos from a authorised channel. You will need to authorise  every channel separately to get auth token for that channel.

**Reply by alibori** [reply]
Thanks @alchemyguy !
